GameData/KerbalismConfig/ConfigTool.cfg

```diff
@@ -0,0 +1,226 @@
+@PART:HAS[@ROKConfigTool]:FINAL
```

Comment:
what does the `@` do when used inside the `:HAS`?

Reply:
`@` looks for a node with the name, similar to how `#` looks for a field